microsoft-edge/web-platform/release-notes/138.md

```diff
@@ -247,7 +246,7 @@ The `<` and `>` characters in HTML element attribute values are now escaped when
 The new `Integrity-Policy` header enables you to assert that every resource of a given type needs to be integrity-checked.  If a resource of the specified type is attempted to be loaded without integrity metadata, that attempt fails, and produces a violation report.
 
 See also:
-* [Subresource Integrity](https://developer.mozilla.org/docs/Web/Security/Subresource_Integrity#browser_compatibility) at MDN.
+* [Browser compatibility](https://developer.mozilla.org/docs/Web/Security/Subresource_Integrity#browser_compatibility) in _Subresource Integrity_ at MDN.
```

Comment:
should this link go to top of page (per orig link text), or to #browser-compatibility section (per orig url anchor?
@captainbrosset 
https://developer.mozilla.org/docs/Web/Security/Subresource_Integrity#browser_compatibility